src/anchor/anchor.scss

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-anchor($status: primary, $underline: 'false') {
+  --_iui-anchor-popout-svg: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'><path d='m16 0v5.4l-1.9-2-8.4 8.4-1.5-1.5 8.3-8.4-1.9-1.9m5.4 16v-9h-1v8h-14v-14h8v-1h-9v16z' /></svg>");
+
+  @include iui-anchor-status($status);
+  border-radius: $iui-border-radius;
+  text-underline-offset: 1px;
+  forced-color-adjust: none;
```

Comment:
This change is anti-accessibility as the colors are no longer "forced high contrast".

Reply:
I'm searching right now to see if there are provided color variables for forced high contrast mode.

The yellow hyperlink color is `hsl(60deg 100% 50%)`.  What if I find a green, orange, red color that match the S & L of the pre-defined yellow?

Reply:
or just remove this line?

the "yellow" comes from Windows settings and can be changed there. That's the whole point of "forced colors" and we shouldn't mess with it except in edge cases like ColorPicker.

These status links are meant to be used in Toast/Anchor which should already have some other indication of the status (i.e. icon), so it's completely fine if the color is lost imo.

Reply:
I still need to figure out how to handle the appended icon.  Here's the options I can think of:

1. `@media (forced-colors: active) { display: none; }`
![Screen Shot 2022-03-29 at 10 38 43 AM](https://user-images.githubusercontent.com/849817/160637094-121bd7c0-2253-41b8-a5d9-f2bf28c55f3a.png)

2. `@media (forced-colors: active) { forced-color-adjust: none; }`
![Screen Shot 2022-03-29 at 10 39 26 AM](https://user-images.githubusercontent.com/849817/160637250-c6ab6c93-508e-4647-a570-3f642f845db3.png)

3. `@media (forced-colors: active) { forced-color-adjust: none; background-color: hsl(60deg, 100, 50); }` but if the user changes their hyperlink color, this will not look great.
![Screen Shot 2022-03-29 at 10 40 31 AM](https://user-images.githubusercontent.com/849817/160637515-6121bcef-f232-482d-95b0-0aa5d3b5b056.png)

Removing `forced-color-adjust: none;` completely results in this which I think we should try to avoid:
![](https://user-images.githubusercontent.com/849817/160484042-330fa044-0333-4b80-ab05-f3208831c456.png)

Reply:
try `@media (forced-colors: active) { background-color: LinkText }` in the pseudo-element

Reply:
in the CSS spec https://www.w3.org/TR/css-color-4/#css-system-colors